### DISCUSSION TITLE
Allow orchestrations to send external events to other orchestrations

### DISCUSSION THREAD

**Discussion by cgillum** [Azure/azure-functions-durable-extension#1780] [Ideas]
Today one can use a `DurableOrchestrationClient` to raise events to a running orchestration instance. However, it would be nice if orchestrations could raise events to other orchestrations as well. Starting with **DurableTask.Core v2.1.0**, this will be possible, and should be exposed to Durable Functions.

**Comment by cfe84** [comment]
Arguably you can just use an activity to do so ; but that would be awesome yes.
